packages/legend-application-studio/src/components/workspace-setup/WorkspaceSetup.tsx

```diff
@@ -158,6 +158,11 @@ export const WorkspaceSetup = withWorkspaceSetupStore(
     const onWorkspaceChange = (val: WorkspaceOption | null): void => {
       if (val) {
         setupStore.changeWorkspace(val.value);
+        if (setupStore.isCurrentProjectNotConfigured) {
```

Comment:
Instead shouldnt we just disable the workspace selector if the project is not configured?